plugin.py

```diff
@@ -37,16 +50,11 @@ def arch() -> str:
         raise RuntimeError("Unknown architecture: " + sublime.arch())
 
 
-def get_setting(view: sublime.View, key: str, default: Optional[Union[str, bool]] = None) -> Union[bool, str]:
+def get_setting(view: sublime.View, key: str, default: Optional[Union[str, bool]] = None) -> Any:
     settings = view.settings()
     if settings.has(key):
         return settings.get(key)
-
-    settings = sublime.load_settings('LSP-rust-analyzer.sublime-settings').get("settings")
-    out = settings.get(key)
-    if out is None:
-        out = default
-    return out
+    return sublime.load_settings('LSP-rust-analyzer.sublime-settings').get("settings", {}).get(key, default)
```

Comment:
```suggestion
    settings = sublime.load_settings('LSP-rust-analyzer.sublime-settings').get("settings", {})
    return settings.get(key, default)
```

This line is too long.

Reply:
I can reorganize this a bit but it's not too long according to the style that we follow in the LSP repo. I've assumed that we can follow the same style here.

See https://github.com/sublimelsp/LSP/blob/main/tox.ini

Reply:
I didn't mean column wise (I also use around the same limit, 125 not 120), I meant in terms of complexity. Chained access like this gets pretty hard to read when it gets excessive.